### PR TITLE
Des 520

### PR DESCRIPTION
**Pull request by wyseguyonline**
Didn't find a whole lot other than a few unnecessary import commands.  Duplicated code is most often due to media queries forcing the redefinition of classes.

### CODE REVIEW

File: scss/elements/templates/_templates.scss [hunk 1]
```diff
@@ -1,6 +1,5 @@
 @import './variables';
 @import './mixins';
-@import '../grid/grid';
```
Comment (JasonLantz) [inline scss/elements/templates/_templates.scss]: good catch....

**Comment by umahaea** [comment]
CI tests pass, so good with QA.

**Comment by umahaea** [comment]
Good to merge.
Regression tests passed: https://travis-ci.org/Pearson-Higher-Ed/ux-test-platform/builds/213574270
sorry for the delay..

**Comment by JasonLantz** [comment]
sorry... i thought i approved this earlier...

**Pull request merged**